CMakeLists.txt

```diff
@@ -523,20 +531,21 @@ endif()
 
 # Cygwin installs the *.dll libraries in bin directory and uses PATH.
 
-set (INSTALL_SHARED_DIR lib)
+set (INSTALL_SHARED_DIR ${CMAKE_INSTALL_LIBDIR})
 if (CYGWIN)
-	set (INSTALL_SHARED_DIR bin)
+	set (INSTALL_SHARED_DIR ${CMAKE_INSTALL_BINDIR})
```

Comment:
Isn't this not set if you dont use GNUInstalldirs?

Reply:
But I do use GNUInstallDirs, unless CMAKE_INSTALL_LIBDIR is set. I state all others are set then as well.